unittests/test_check_filters.py

```diff
@@ -37,51 +37,56 @@ def setUp(self):
                 'num_gpus_per_node': 1})
         ]
 
+    def count_checks(self, filter_fn):
+        return sn.count(filter(filter_fn, self.checks))
+
     def test_have_name(self):
-        self.assertEqual(1, sn.count(filter(filters.have_name('check1'),
-                                            self.checks)))
-        self.assertEqual(1, sn.count(filter(filters.have_name('check2'),
-                                            self.checks)))
-        self.assertEqual(1, sn.count(filter(filters.have_name('check3'),
-                                            self.checks)))
-        self.assertEqual(0, sn.count(filter(filters.have_name('check4'),
-                                            self.checks)))
+        self.assertEqual(1, self.count_checks(filters.have_name('check1')))
+        self.assertEqual(3, self.count_checks(filters.have_name('check')))
+        self.assertEqual(2, self.count_checks(filters.have_name('\S*1|\S*3')))
+        self.assertEqual(0, self.count_checks(filters.have_name('Check')))
```

Comment:
Change to raw string `r'\S*1|\S*3'` so that the pycodestyle does not complain.

Reply:
Can you do it as a suggestion through Github, so that we can do the change on-the-fly?